src/httpClient.ts

```diff
@@ -19,7 +19,7 @@ export default class httpClient {
     }
 
     async getAirtableRecords() {
-        const response = await this.httpGet('http://localhost:5000/airtable_scraper/records');
+        const response = await this.httpGet('/airtable_scraper/records');
```

Comment:
Let's set the url prefix (i.e. `http://localhost:5000`) as an environment variable and run a substitution in the generic httpGet method.

Reply:
I circumvented the cors issue by creating a proxy in the package.json

Reply:
ahh yeah I see that now. maybe next thing to look at is parameterizing that somehow (with env var or something) to ease deployment